Application/EdFi.Ods.Tests/EdFi.Ods.Security/Authorization/EdFiAuthorizationProviderTests.cs

```diff
@@ -890,7 +890,7 @@ public void
                 Should_resolve_claims_using_authorization_metadata_order_rather_than_callers_claims_order_and_invoke_lowest_matching_claims_authorization_strategy()
             {
                 SecondAuthorizationStrategy.SingleItemWasCalled.ShouldBeTrue();
-                FourthAuthorizationStrategy.SingleItemWasCalled.ShouldBeFalse();
+                FourthAuthorizationStrategy.SingleItemWasCalled.ShouldBeTrue();
```

Comment:
NOTE TO SELF: This behavior verified by this test looks suspicious. Needs specific review.

Reply:
I revert this change